packages/iTwinUI-react/src/core/Tile/Tile.tsx

```diff
@@ -141,6 +158,31 @@ export const Tile = (props: TileProps) => {
   const showMenu = React.useCallback(() => setIsMenuVisible(true), []);
   const hideMenu = React.useCallback(() => setIsMenuVisible(false), []);
 
+  const renderTitleIcon = () => {
+    if (!!titleIcon) {
+      return React.cloneElement(titleIcon, {
```

Comment:
I tried adding
```ts
<span className='iui-tile-status-icon' aria-hidden>
   {titleIcon}
</span>
```

But styling does not work perfectly when titleIcon is ProgressRadial (which is one of the main items to add there).